toc.md

```diff
@@ -742,12 +743,12 @@ module Workerholic
 end
 ```
 
-When Workerholic starts, it'll load the app which load Rails if it detects that it's working a Rails application, and then we require our own active job adapter and load that along with the rest of the Rails application.
+When Workerholic starts, it loads the main application in order to have access to the job classes. This way it can perform the jobs on the processing side. In case a Rails app is detected it will load the Rails application codebase along with our own active job adapter.
 
 ### Testing
-As we developed our features, we needed to test our code.
+As we were developing our features, we needed to test our code.
```

Comment:
I'd switch the POV here:
> When building a BJP, you might want to introduce some testing to make sure you are not breaking any part of your BJP by adding a feature or refactoring your code.
> For Workerhollic, we decided to have a testing environment separated from our development environment by setting a Redis DB on a different port. We also flush Redis before each run to ensure that it provides a valid state for every test case.

(we do flush BEFORE, not after, as shown in the code snippet).

I'd move the flushing part right before the correct code snippet instead of just before all code snippets.